docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -0,0 +1,111 @@
+# Scheduler
+
+![image](EdgeX_SupportingServicesScheduling.png)
+
+## Introduction
```

Comment:
Please add an `!!! edgey` note that states what is new in the Scheduler for 2.0. One would be the revamped API using DTOs. Here is an example of an  `!!! edgey` note: 
https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/microservices/application/Triggers.md
Renders like this:
![image](https://user-images.githubusercontent.com/44779287/124190851-cb32ce80-da77-11eb-95f8-68e5e6ed2e0e.png)

Reply:
Updated.